packages/config/config/devices/0x0312/mp22z.json

```diff
@@ -19,6 +19,21 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 3,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "BASIC SET",
```

Comment:
Note that the manual does not explain what the difference is between these two groups.